overrides/android-override.json

```diff
@@ -483,6 +483,18 @@
             },
             "minSupportedVersion": 51670000
         },
+        "privacyDashboard": {
+            "state": "enabled",
+            "features": {
+                "highlightedProtectionsToggle": {
```

Comment:
Move this section to the feature file you created. If other platforms adopt this feature they can override the features or settings as needed. In this override you can just leave the `"state": "enabled"` property to ensure this is only enabled on Android.